elasticdl/python/worker/worker.py

```diff
@@ -366,11 +365,12 @@ def training_process_eagerly(self, features, labels):
         grads = tape.gradient(loss, self.get_trainable_items())
         return loss, grads
 
+    # TODO (yunjian.lmh): `evaluation_process` and `predict_process` are the
+    # same now.
```

Comment:
Why isn't "labels" used?

Reply:
Because workers report model outputs and labels to the master, and the master calculating evaluation metrics.